front/src/common/validators/minimum-star-number.ts

```diff
@@ -0,0 +1,32 @@
+import { FieldValidationResult } from 'lc-form-validation';
+
+export const VALIDATION_TYPE = 'FIELDS_MATCH';
+
+
+export const validateMinimumStarNumber = (value: number, vm: any, key: string): FieldValidationResult => {
```

Comment:
Let's make it more generic, we are checking that the number of stars must be at least three.

What we are checking here is that a given user has entered number and is greater than 3, we can decouple it from the domain.
```typescript
export const numberIsGreaterThan(value, vm, customParams) : FieldValidationResult =>
```

On the other hand